Jumpscale/data/nacl/NACL.py

```diff
@@ -151,30 +150,19 @@ def _hash(self, data):
         m.update(data)
         return m.digest()
 
-    @property
-    def privkey(self):
-        if self._privkey == "":
-            self._privkey = self.file_read_hex(self.path_privatekey)
-        key = self.decryptSymmetric(self._privkey)
-        privkey = PrivateKey(key)
-        self._pubkey = privkey.public_key
-        return privkey
-
     @property
     def words(self):
         """
         js_shell 'print(j.data.nacl.default.words)'
         """
-        privkey = self.privkey.encode()
-        return j.data.encryption.mnemonic.to_mnemonic(privkey)
-        # if not j.sal.fs.exists(self.path_words):
-        #     self._log_info("GENERATED words")
-        #     words = j.data.encryption.mnemonic_generate()
-        #     words = self.encryptSymmetric(words)
-        #     self.file_write_hex(self.path_words,words)
-        # words = self.file_read_hex(self.path_words)
-        # words = self.decryptSymmetric(words)
-        # return words.decode()
+        raw_key = self.signingkey.encode()
```

Comment:
Is there a good reason why we would not keep using `self.privkey.encode()` here, @zaibon?

Reply:
cause we want the mnemonic workd to be the seed of everything. And since now we get everything out of the signing key, we need to use the signing key to generate the mnemonic

Reply:
Got it, perfect.